website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -110,138 +115,159 @@ Here is the current architecture of our application:
   </tr>
 </table>
 
-
-
 ## The html service folder
 
-The html service is located in a subdirectory of the application,
-in `~/todo-app/html-server/`.
-This makes sense because it is an integral part of our application,
-and doesn't make sense outside of it.
+The html service is located in a subdirectory of the application, in
+`~/todo-app/web-service/`. This makes sense because it is an integral part of
+our application, and doesn't make sense outside of it.
 
-Most of the files in this folder
-are just a normal [ExpressJS](http://expressjs.com) application,
-plus some extra tools like linters.
+Most of the files in this folder are just a normal
+[ExpressJS](http://expressjs.com) application, plus some extra tools like
+linters.
 
 Let's check out how the service looks like internally.
 
-
 ### The service configuration file
 
-This file tells the Exosphere framework everything it needs to know about this service.
+This file tells the Exosphere framework everything it needs to know about this
+service.
 
-__~/todo-app/html-server/service.yml__
+**~/todo-app/web-service/service.yml**
```

Comment:
Why do you change `__` to `**` here? It doesn't really change anything.

Reply:
I think this was the formatter. I don't remember consciously changing this nor do I have a good reason if I did. Either way, I will blame the formatter and correct it.